javascript/index.ts

```diff
@@ -1,58 +1,64 @@
-import {AlreadyExistsError, CacheGetStatus, LogLevel, LogFormat, SimpleCacheClient,} from '@gomomento/sdk';
+import {
+  AlreadyExistsError,
+  CacheGetStatus,
+  LogLevel,
+  LogFormat,
+  SimpleCacheClient,
+} from '@gomomento/sdk';
 
-const cacheName = "cache";
-const cacheKey = "key";
-const cacheValue = "value";
+const cacheName = 'cache';
+const cacheKey = 'key';
+const cacheValue = 'value';
 const ttl = 60;
 const authToken = process.env.MOMENTO_AUTH_TOKEN;
 if (!authToken) {
-  throw new Error("Missing required environment variable MOMENTO_AUTH_TOKEN");
+  throw new Error('Missing required environment variable MOMENTO_AUTH_TOKEN');
 }
 
 const defaultTtl = 60;
 const momento = new SimpleCacheClient(authToken, defaultTtl, {
   loggerOptions: {
     level: LogLevel.DEBUG,
-    format: LogFormat.CONSOLE
-  }
+    format: LogFormat.CONSOLE,
+  },
 });
 
 const main = async () => {
   try {
     await momento.createCache(cacheName);
   } catch (e) {
     if (e instanceof AlreadyExistsError) {
-      console.log("cache already exists");
+      console.log('cache already exists');
     } else {
       throw e;
     }
   }
 
-  console.log("Listing caches:");
+  console.log('Listing caches:');
```

Comment:
maybe for another pr but just seeing this now I think maybe these log messages are redundant with debug logging we have in SDK now. Maybe better to remove these since we configure `level: LogLevel.DEBUG,` explicitly up top.